### DISCUSSION TITLE
How to run a file that exists in node_modules with pnp?

### DISCUSSION THREAD

**Discussion by amirhhashemi** [yarnpkg/berry#3481] [Q&A] [answered]
I'm using TypeOrm in my application and it has CLI that I need it. The CLI doesn't work with typescript and if I want to use it I have to add this script to the package.json file:
```
"scripts": {
    ...
    "typeorm": "node --require ts-node/register ./node_modules/typeorm/cli.js"    
}
```
and now I can run this command:
`yarn run typeorm migration:run`

I recently switched to yarn 3 and I'm using Plug'n'Play. so there is no node_modules directory and running the above command gives me some errors. I need a way to run the `./node_modules/typeorm/cli.js` file. something like this:
`"typeorm": "yarn node --require ts-node/register typeorm/cli.js" `

Is there a way to do it without creating the node_modules directory?

**Comment by merceyz** [comment, marked as answer]
You can use [`yarn bin`](https://yarnpkg.com/cli/bin) to get the location of their CLI
```json
{
	"scripts": {
		"typeorm": "node --require ts-node/register \"$(yarn bin typeorm)\""
	}
}
```

**Reply by amirhhashemi** [reply]
Thank you. it works perfectly.

**Reply by mgohin** [reply]
You saved me :)
